src/main/java/fi/metatavu/soteapi/system/SystemHealthCheck.java

```diff
@@ -0,0 +1,37 @@
+package fi.metatavu.soteapi.system;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Initialized;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+
+import fi.metatavu.soteapi.settings.RunMode;
+import fi.metatavu.soteapi.settings.SystemSettingController;
+
+@ApplicationScoped
+public class SystemHealthCheck {
+
+  @Inject
+  private Logger logger;
+  
+  @Inject
+  private SystemSettingController systemSettingController;
+
+  public void init(@Observes @Initialized(ApplicationScoped.class) Object init) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "init". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)